packages/labs/ssr/src/lib/render-global.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import './dom-shim-global.js';
+export * from './render-lit-html.js';
```

Comment:
I don't think this module or `dom-shim-global` are needed. Because we mutate the global, I'd rather have the importer call the install function so they know they're doing it. It's not much code.

Reply:
Oh, so I forgot, there's actually a practical issue here related to module ordering, which is what led me down this path in the first place. Because all the module imports get hoisted, you can't import and run `installGlobalDomShim()` _and_ load your component definitions in the same module. So the user would always need to make their own intermediate module like this, just to be able to do "import render & components" in the same module, which should be very common.

I went ahead with:
* `install-global-dom-shim.js`
* `render-with-global-dom-shim.js`

Pretty ugly, but let me know if the above reason makes sense and if it makes you want to have less unwieldily names as a result.